spec/javascripts/app/views/location_stream_spec.js

```diff
@@ -3,7 +3,7 @@ describe("app.views.LocationStream", function() {
     this.post = factory.post();
     this.view = new app.views.LocationStream({model : this.post});
     /* jshint camelcase: false */
-    gon.appConfig = {map: { mapbox: {enabled: true, id: "yourID", access_token: "yourAccessToken" }}};
+    gon.appConfig = {map: { mapbox: {enabled: true, access_token: "yourAccessToken" }}};
     /* jshint camelcase: true */
```

Comment:
- There should be no space after '{'
- Identifier 'access_token' is not in camel case.
- There should be no space before '}'